assessments/guide/self-assessment.md

```diff
@@ -54,8 +56,30 @@ multiple projects.
 Provide information for reviewers who may not be familiar with your project's
 domain or problem area.
 
-### Goal
-The intended goal of the projects including the security guarantees the  project
+### Actors
+These are the individual parts of your system that interact to provide the 
+desired functionality.  Actors only need to be separate, if they are isolated
+in some way.  For example, if a service has a database and a front-end API, but
+if a vulnerability in either one would compromise the other, then the distinction
+between the database and front-end are not relevant.
+
+The means by which actors are isolated should also be described, as this is often
+what prevents an attacker from moving laterally after a compromise.
```

Comment:
like actions, if you can add an example definition to illustrate, that would be useful. We should provide a table for them to fill for actors/actions to help provide some guidelines.. like `actor | description | isolation`, and for actions maybe a ` action | actor(s) | description | security considerations`

Reply:
The rest of this document is pretty sparse.  (I'm actually adding a lot of detail, in comparison.)  Do you think it belongs here or in an example assessment for them to look at?

Reply:
yea agreed - looks like that’s a gap! Can you help open a ticket to add the the self-assessment template? lgtm this.